### PR TITLE
Fix some grammar and typos in the first tenth of the monsters.json file (English)

### PR DESCRIPTION
**Pull request by krislion**
What grammatical principle reigns in these descriptions? (monsters.json)

When viewing the descriptions via "x"amine in-game, I often get weird partial sentences with occasional random spelling.
In this first commit, I open myself to discussion by converting a few of these to sentences.

If this seems like a good direction, I will go on to fix a few more instances, including several issues where "it's" is used in place of "its" and several issues with comma splices.

Since this project uses *.po files, I understand this should only merge after updating the checked-in *.po files as well by running the related scripts.

Nice to make your acquaintance.

### CODE REVIEW

**Comment by illi-kun** [comment]
> Since this project uses *.po files, I understand this should only merge after updating the checked-in *.po files as well by running the related scripts.

No need to run the mentioned scripts / update any files for translators. Your PR is mergeable as-is.

File: data/json/monsters.json [hunk 1]
```diff
@@ -210,7 +210,7 @@
     "vision_day":40,
     "hp":16,
     "death_function":"NORMAL",
-    "description":"A spotted wild cat living across much of North America.  It is not a serious threat to humans, but it can be aggressive if not left alone.",
+    "description":"A spotted wild cat, these live across much of North America.  It is not a serious threat to humans, but it can be aggressive if not left alone.",
```
Comment (illi-kun) [inline data/json/monsters.json]: ```"cat ... these live"``` confuses me a bit (singular & plural forms in one sentence). I'm not a native speaker, so maybe it's my mistake and sentence is actually correct, so just let me know if everything is okay here.

Comment (Rivet-the-Zombie) [inline data/json/monsters.json]: I think it's *technically* correct, but it's still a bit of odd phrasing.

Comment (krislion) [inline data/json/monsters.json]: The original phrase may imply the specific bobcat the character encounters has been going abroad across North America, which does not make sense. One of my goals is to reduce confusion; so, I will revise the trickier part.

I will make a new commit tomorrow with a simpler phrase on bobcat entry.
I will also transition more of the phrases to sentences in the monsters.json file.

Thanks for the quick turnaround.

Comment (i2amroy) [inline data/json/monsters.json]: Alternatively you could replace the comma with a semi-colon which might make it a bit more clear about the division in the phrasing, i.e.: "A spotted wild cat; these live across much of North America".

**Comment by illi-kun** [comment]
@krislion are you interested in finalizing this PR?

**Comment by mugling** [comment]
Closed as appears stalled. Can be opened if OP wants to resume work on it

**Pull request closed without merging**